.github/workflows/build.yaml

```diff
@@ -0,0 +1,66 @@
+name: Build and Push Image for Multi-Arch CI- Additional support for s390x & Power along with arm64 & amd64
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - 'release-[0-9]+\.[0-9]+\.[0-9]+'
+    tags:
+      - 'latest'
+      - 'v*'
+
+
+env:
+  IMAGE_NAME: "odh-model-controller"
+  REPO_NAME: "opendatahub"
+  REGISTRY_NAME: "quay.io"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v3
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_ACCESS_TOKEN }}
+
+      - name: Set Docker Tag
+        run: |
+          if [[ "${GITHUB_REF}" == "refs/heads/main" ]]; then
+            echo "TAG=latest" >> $GITHUB_ENV
+          elif [[ "${GITHUB_REF}" == refs/heads/release-* ]]; then
+            VERSION="${GITHUB_REF#refs/heads/release-}"
+            echo "TAG=v${VERSION}" >> $GITHUB_ENV
```

Comment:
are you happy with `vX.Y.Z`? or would you rather `vX.Y.Z-latest`?

Reply:
@israel-hdez We have stick to 2 different type of tags i.e. currently available for x86 on quay repository 

- Release Tag ( v0.11.0) 
- latest 

Please suggest how did you want us to handle it. Should we stick to the same naming convention for tag or the one which you have suggested in comments ie vX.Y.Z-latest

Reply:
I was checking our prow/tide configs and it's using the vX.Y.Z tags. So, it is a conflict and this workflow and prow/tide can overwrite the other image, depending on which one push at last.

It will be better to use a -latest suffix, or some other suffix to prevent the conflict.

Reply:
ok

Reply:
Hi @israel-hdez, we've made the changes as requested to add a suffix to the tag (vX.Y.Z-latest). Please help to review.